core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -18,6 +21,36 @@ import kotlin.reflect.typeOf
 
 // region DataColumn
 
+/**
+ * ## The CumSum Operation
+ *
+ * Computes the cumulative sum of the values in the {@get DATA_TYPE}.
+ *
+ * __NOTE:__ If the column contains nullable values and `skipNA` is set to `true`,
```

Comment:
I'd write "`` `skipNA` ``" like "`[skipNA\]`" so users can click on it in the kdocs :)